behave_test_automation/pah_behave/features/steps/common_steps.py

```diff
@@ -0,0 +1,18 @@
+from behave import given, then, when
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+@given(u'User opens pah-fm website')
+def step_impl(context):
```

Comment:
To be honest, name doesn't say me what this function do.

Reply:
Changed the name to: open_main_url